### ISSUE TITLE
CKAN 2.5a: db_init requires extend_existing=True

### ISSUE THREAD

**Issue by florianm** [ckan/ckanext-pages#27]
Reported also at ckan/ckan#2679 for reference.

Running into 
```
...
(ckanext-pages gets called for the first time)
File '/project/ckanext-pages/ckanext/pages/actions.py', line 223 in pages_list
  return _pages_list(context, data_dict)
File '/project/ckanext-pages/ckanext/pages/actions.py', line 76 in _pages_list
  db.init_db(context['model'])
File '/project/ckanext-pages/ckanext/pages/db.py', line 112 in init_db
  sa.Column('extras', types.UnicodeText, default=u'{}'),
File '/usr/lib/ckan/local/lib/python2.7/site-packages/sqlalchemy/sql/schema.py', line 339 in __new__
  "existing Table object." % key)
InvalidRequestError: Table 'ckanext_pages' is already defined for this MetaData instance.  Specify 'extend_existing=True' to redefine options and columns on an existing Table object.
```

Can be fixed with adding "extend_existing" to `ckanext-pages/ckanext/pages/db.py` in function `init_db()` after the the sa.Columns as shown:
```
    pages_table = sa.Table('ckanext_pages', model.meta.metadata,
        sa.Column('id', types.UnicodeText, primary_key=True, default=make_uuid),
        sa.Column('title', types.UnicodeText, default=u''),
        sa.Column('name', types.UnicodeText, default=u''),
        sa.Column('content', types.UnicodeText, default=u''),
        sa.Column('lang', types.UnicodeText, default=u''),
        sa.Column('order', types.UnicodeText, default=u''),
        sa.Column('private',types.Boolean,default=True),
        sa.Column('group_id', types.UnicodeText, default=None),
        sa.Column('user_id', types.UnicodeText, default=u''),
        sa.Column('publish_date', types.DateTime),
        sa.Column('page_type', types.DateTime),
        sa.Column('created', types.DateTime, default=datetime.datetime.utcnow),
        sa.Column('modified', types.DateTime, default=datetime.datetime.utcnow),
        sa.Column('extras', types.UnicodeText, default=u'{}'),
        extend_existing=True
    )
```
followed by `datacats install` (if using datacats) or presumably a `python setup.py install` in a source install.

Seems to work, haven't run any tests though apart from making sure pages work in GUI.

**Comment by SaintlyVi** [comment]
Thanks so much for sharing! Amazing to find a fix that works instantaneously.

**Comment by metaodi** [comment]
Fixed with #28
